cnxarchive/tests/views/test_content.py

```diff
@@ -1166,6 +1244,7 @@ def test_extra_w_utf8_characters(self):
         self.assertEqual(self.request.response.content_type,
                          'application/json')
         output['canPublish'].sort()
+        output['books'].sort()
```

Comment:
This test started failing for no reason, with the books coming in the opposite order. The `get-books-containing-page` code in cnxdb does sort the books, but these books have the same exact authors and revised dates, so I think the test was always flaky. I started sorting the books here and made the expected values match.